event-planner-frontend/src/features/admin-management/store/thunks/createUserOrAdminThunk.ts

```diff
@@ -0,0 +1,14 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import { UserOrAdminDto } from "features/admin-management/api/dtos";
+import UserService from "services/userService";
+
+const userService: UserService = new UserService();
+
+export const createUserOrAdminThunk = createAsyncThunk("post/createUserOrAdmin", async (requestData: UserOrAdminDto, thunkAPI) => {
```

Comment:
This is the logical structure of every thunk name, it can be leveraged to determine where the async thunk will be user
```suggestion
export const createUserOrAdminThunk = createAsyncThunk("adminSlice/createUserOrAdmin", async (requestData: UserOrAdminDto, thunkAPI) => {
```